From c22ba4e35bc12ca56296a97f6f3468106804ad30 Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 11 Apr 2020 18:54:33 -0700
Subject: [PATCH 10/26] avcodec/mips: fix get_cabac_inline_mips function name

On other platforms, the functions are named get_cabac_inline_xxx but not
this one. There's also a define.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
Signed-off-by: Michael Niedermayer <michael@niedermayer.cc>
---
 libavcodec/mips/cabac.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavcodec/mips/cabac.h b/libavcodec/mips/cabac.h
index 2a05e5ab3c..03b5010edc 100644
--- a/libavcodec/mips/cabac.h
+++ b/libavcodec/mips/cabac.h
@@ -29,7 +29,7 @@
 #include "config.h"
 
 #define get_cabac_inline get_cabac_inline_mips
-static av_always_inline int get_cabac_inline(CABACContext *c,
+static av_always_inline int get_cabac_inline_mips(CABACContext *c,
                                              uint8_t * const state){
     mips_reg tmp0, tmp1, tmp2, bit;
 
-- 
2.30.1

